# Default configuration for emergency handler
---
/**:
  ros__parameters:
    update_rate: 10
    timeout_hazard_status: 0.5
    use_parking_after_stopped: false
    use_comfortable_stop: false

    # setting whether to turn hazard lamp on for each situation
    turning_hazard_on:
      emergency: true
